miniapps/shifted/distance.cpp

```diff
@@ -72,7 +72,7 @@
 //     mpirun -np 4 distance -m ./corners.mesh -p 0 -rs 3 -t 200.0
 //
 //   Problem 1: zero level set: ball at the center of the domain - the exact
-//   distance is known, the code computes global and local errors.
+// + distance is known, the code computes global and local errors.
```

Comment:
very minor: is the `+` intended ? I have not seen it in other places to indicate continuation of comment over a line break. e.g. `meshing/pminimal-surface.cpp`  uses an intend.

Reply:
This is needed to prevent this line from being detected as a sample-run by the script which basically searches for two spaces followed by the name of the example: https://github.com/mfem/mfem/blob/110bec150762d363452c16c72cea7ab3a3bc6ec7/config/sample-runs.sh#L313-L317

The current logic for searching for sample-runs is a little vulnerable to cases like the above. We probably should consider a different convention in a future PR, e.g. use lines starting with `//>` (or something like that) as sample-runs.

Reply:
thanks for the clarification. Yes, makes sense.